src/scancode/resource.py

```diff
@@ -1392,7 +1420,8 @@ def __init__(self, location,
                  codebase_attributes=None,
                  full_root=False, strip_root=False,
                  temp_dir=temp_dir,
-                 max_in_memory=10000):
+                 max_in_memory=10000,
+                 depth=-1):
```

Comment:
What about using `**kwargs` instead and not doing anything with it below?

Reply:
Yeah, that would be neater I think.